packages/promotion/src/utils/compute-actions/shipping-methods.ts

```diff
@@ -95,14 +99,18 @@ export function applyPromotionToShippingMethods(
     const totalApplicableValue = shippingMethods!.reduce((acc, method) => {
       const appliedPromoValue = methodIdPromoValueMap.get(method.id) || 0
 
-      return acc + method.subtotal - appliedPromoValue
+      return acc + (method.subtotal || 0) - appliedPromoValue
```

Comment:
I know it doesn't matter in this case, but it's nice practice to rely on `??` instead :)